test/functional/rpc_rawtransaction.py

```diff
@@ -24,8 +24,8 @@
 from test_framework.util import (
     assert_equal,
     assert_raises_rpc_error,
-    find_vout_for_address,
 )
+from test_framework.wallet import MiniWallet, getnewdestination
```

Comment:
nit:
```suggestion
from test_framework.wallet import (
    MiniWallet,
    getnewdestination,
)
```
(that way, potential merge conflicts can be resolved easier)